lib/model/model.go

```diff
@@ -1559,18 +1559,38 @@ func sendIndexTo(minSequence int64, conn protocol.Connection, folder string, fs
 	return maxSequence, err
 }
 
+func (m *Model) makeUsLastModified(fs []protocol.FileInfo) []protocol.FileInfo {
+	fsMod := make([]protocol.FileInfo, 0, len(fs))
+	for _, file := range fs {
+		file.ModifiedBy = uint64(m.id.Short())
```

Comment:
The corresponding place for deletes is here:

https://github.com/syncthing/syncthing/blob/master/lib/model/model.go#L1873

We should probably have methods on FileInfo for doing this instead, i.e.,

```
func (f FileInfo) Deleted(by ShortID) FileInfo
func (f FileInfo) Updated(by, newBlocks, newSize, newModtime etc)
```

Future improvement!